app/assets/stylesheets/splash.css

```diff
@@ -0,0 +1,129 @@
+
+.main_splash {

```

Comment:
Great job @ogiogiovictor 🤝

- [ ] As stated in the project requirements, kindly follow the design guidelines so that your project can replicate the design given. Make sure your button is similar to the button of the given design.
![design-behance](https://user-images.githubusercontent.com/63917725/209407972-9ee57593-4fab-4fb8-9caf-0fe4b4dc82ae.PNG)